include/nbl/builtin/hlsl/shapes/beziers.hlsl

```diff
@@ -2,741 +2,355 @@
 // This file is part of the "Nabla Engine".
 // For conditions of distribution and use, see copyright notice in nabla.h
 
+#ifndef _NBL_BUILTIN_HLSL_SHAPES_BEZIERS_INCLUDED_
+#define _NBL_BUILTIN_HLSL_SHAPES_BEZIERS_INCLUDED_
+
 // TODO: Later include from correct hlsl header
 #ifndef nbl_hlsl_FLT_EPSILON
 #define	nbl_hlsl_FLT_EPSILON 5.96046447754e-08
 #endif
```

Comment:
gonna be the `numeric_limits` by @atkurtul

Reply:
use this if integrated, if not add comment //TODO: